docs/user-guide/index.md

```diff
@@ -0,0 +1,154 @@
+# User guide
+
+## Getting started
+
+
+::::{grid} 2
```

Comment:
How does it look if we just arrange those 3 cards on a single line?

Reply:
It would work. I wanted to give those items more weight because they are more important for newcomers. But I don't have a strong opinion. Should I change?

Reply:
Change and we can see how it looks? But I think it will look better?

Reply:
Done:
![guide](https://github.com/scipp/plopp/assets/11393224/71c804a7-38c4-430e-8602-668653c58e79)